backend/app.py

```diff
@@ -34,25 +36,55 @@ def get_surveys():
     data = db.executeSelectQuery(query)
     response = {}
     for d in data:
-        title, name, desc, s_id, q_id, q_text, qnumber, c_val, cnumber = d
-        pass
-
-    response = {
-        "Surveys" : [
-            {
-            "Id": None,
-            "Title": None,
-            "Name": None,
-            "Description": None,
-            "Question" : [{
-                "QNumber": None,
-                "Text": None,
-                "Choices" : [{
-                    "CNumber": None,
-                    "Value": None,
-                }]
-            }]
+        title, name, desc, s_id, qnumber, q_text, c_val, cnumber = d
+        if s_id not in response.keys():
+            response[s_id] = {
+                "Title": title,
```

Comment:
nit: you dont have to map these to the "Title". it's generally common practice to return the json object with camelcase or snake_case.

Reply:
Hey Jin, sorry for my confusion, what do you mean here? Is the suggestion to change the key "Title" (survey's title)?

Reply:
yup the response has a key of `Title`, this can just be `title`. I also could be wrong with how python does it but maybe object can just be 
```
{
   title,
   name,
   desc,
   questions
}

and I think that would be set as key title with value of the title filled in (this works in js).

Reply:
Ohhh I see. I think I need a key in Python, but I could look into building the json without it, if this is an issue when you are parsing.

Reply:
ahh got it! it's really not a big deal I am just defining type for the response atm and I wanted to finalize the key names.

Reply:
Gotcha! Would it be a problem if we kept like this? Otherwise, I can spend some time looking into it no problem.

Reply:
Actually now that I think of it, can you actually return the outer data structure as an array with nested objects inside? so it would be like 
```
[
  Description: ...,
  Name: ...,
  Question: [
    Choices: [
        // string of values here
      ]
  ]
```

I realized all the keys here are just sorted in order so returning them as arrays would be preferable.

If this sounds like too much work let me know and I will massage the data on the front end side.

Reply:
also the key names capitalized is fine I will just make those are the key names in my type file or I can simply convert them when I get the response back

Reply:
So, just to double check, the change would be to convert the outer dictionary into an array and the inner for Choices as well, correct? For the former, I understand and I could add a key for the survey id because we need that for the second endpoint. For the latter, we would be missing the choice numbers, which is also important for matching (second endpoint), but you could give me that based on the index of the choice too.

Reply:
You are correct here, there is no guarantee that the surveyId and the choice # is in order and doesn't skip indicies, let's keep data structure as is and I will work with it on the front end 👍

Reply:
Lmk if this becomes a problem tho, and i will be happy to change it :)

Reply:
its actually not, im changing my structure on frontend for how i handle this, initially I was gonna leverage next's static generation related functions but I realized we are actually missing a couple endpoints on that and I really dont want us to go through the extra work of creating those and get it hooked up.
I am going to basically change the structure on frontend to utilize context api to make one initial fetch to the surveys (since it returns it all), memoize it and use the context provider to supply that data to every single page that wants access to it. this solution should work just fine

Reply:
sounds good, any more suggestions before I merge it? Thanks for the review again Jin!